app/controllers/race_editions_controller.rb

```diff
@@ -54,11 +55,16 @@ def race_entries
   def create_entry
     @racer = Racer.new(obj_params[:racers_attributes]['0'])
 
-    if @racer.save
-      race_entry = RaceEntry.create(race_edition: @race_edition, racer: @racer)
+    default_race_entry_attributes = {race_edition: @race_edition, racer: @racer}
+    provided_race_entry_attributes = obj_params[:race_entries_attributes]['0'] || {}
```

Comment:
This looks strange to me. What's the ['0']? That looks ugly. I'd want either a better way to write this or a comment explaining.